t/50mruby-channel.t

```diff
@@ -0,0 +1,95 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use t::Util;
+
+plan skip_all => 'mruby support is off'
+    unless server_features()->{mruby};
+
+subtest "process integer" => sub {
+    my $server = spawn_h2o(<< 'EOT');
+hosts:
+  default:
+    paths:
+      /:
+        mruby.handler: |
+          Proc.new do |env|
+            ch = create_channel
+            ch.push 1
+            res = ch.shift
+            [200, {}, [res]]
+          end
+
+EOT
+    my ($stderr, $stdout) = run_prog("curl --silent --dump-header /dev/stderr http://127.0.0.1:$server->{port}/");
+    is $stdout, 1;
+};
+
+plan skip_all => 'mruby support is off'
```

Comment:
JFYI, you don't have to call `skip_all` multiple times

Reply:
I deleted needless `skip_all `
https://github.com/h2o/h2o/pull/1336/commits/8f4c9bf1e33ec15db341eccd52b94ead21f32267